docs/DeveloperGuide.md

```diff
@@ -899,51 +970,145 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
     Use case ends.
 
-**Use case: UC13 View Help**
+**Use case: UC13 Remove a macro**
 
 **MSS**
-1. User asks for the list of command.
-2. CLI-nic displays information about all the commands and contains sample commands that the user can try out.
-3. User try out commands listed under help to familiarise themselves with CLI-nic.
+
+1. User keys in command to remove a macro with a specified alias.
+2. CLI-nic removes the macro and shows a success message.
 
     Use case ends.
 
 **Extensions**
 
-* 1a. User asks for a specific command via the command keyword.
-  * 1a1. CLI-nic displays instruction on how the command they ask can be used. Sample command call will also be given.
-  * 1a2. User follows the instruction to try out command they asked to familiarise themselves with this command.
+* 1a. The alias supplied is invalid.
+
+  * 1a1. CLI-nic shows an error message with the constraints for the alias specified.
+  * 1a2. User enters a new command.
+
+    Steps 1a1-1a2 are repeated until the alias entered is valid. <br>
+    Use case resumes from step 2.
+
+* 1b. The command supplied is not in the correct format.
+
+  * 1b1. CLI-nic shows an error message and the correct format, along with examples of the correct usage.
+  * 1b2. User enters a new command.
+
+    Steps 1b1-1b2 are repeated until the command format is correct. <br>
+    Use case resumes from step 2.
+
+* 1c. An existing macro with the alias supplied cannot be found.
+
+  * 1c1. CLI-nic shows an error message, notifying the user that the macro does not exist.
+  * 1c2. User enters a new command with another alias.
+
+    Steps 1c1-1c2 are repeated until a macro with the new alias supplied is found. <br>
+    Use case resumes from step 2.
+
+**Use case: UC15 Update a Product in a Supplier**
```

Comment:
Should we split this to 2 test cases, one where product exist, another it does not exist (So that the if statement does not shows up in step 2 of MSS?)